extensions/scala/tests/src/test/scala/org/virtuslab/ideprobe/scala/ScalaTestRunConfigurationTest.scala

```diff
@@ -24,7 +24,13 @@ class ScalaTestRunConfigurationTest extends ScalaPluginTestSuite {
 
     val moduleRunConfiguration = ScalaTestRunConfiguration.Module(moduleRef)
     val moduleRunResult = intelliJ.probe.run(moduleRunConfiguration)
+    val allTests = moduleRunResult.suites.flatMap(_.tests).map(_.status)
+    val failedTests = allTests.filter(_.isInstanceOf[org.virtuslab.ideprobe.protocol.TestStatus.Failed])
```

Comment:
if you need to do such thing, it is probably signal that our api is lacking. You could either add testCount, failedTestCount, ignoredTestCount to the run result, or add .isFailed to test status. Or both.

Reply:
Good point, but I'm not going to do that in this PR if I don't have to add new tests.